care/users/models.py

```
@@ -311,12 +311,12 @@ class User(AbstractUser):
     )
 
     verified = models.BooleanField(default=False)
-    deleted = models.BooleanField(default=False)
+    deleted = models.BooleanField(default(False))
```

Ruff (0.8.2)

Undefined name `default`

(F821)